src/lib/types.ts

```diff
@@ -106,5 +108,12 @@ export type Bid = {
 	amount: number;
 	bidderAccount: string;
 	date: Date;
-	tokenId?: string;
+	tokenId: number;
```

Comment:
Token ID is a `uint256` which is larger than a JS `number` can hold, this should be `string` instead